docs/jotai/api/babel.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Babel
+description: This doc describes the `jotai/babel` bundle.
+nav: 2.4
+---
+
+## plugin-debug-label
+
+Jotai is based on object references and not keys (like Recoil). This means there's no identifier for atoms. To identify atoms, it's possible to add a `debugLabel` to an atom, which can be found in React devtools.
```

Comment:
I'm not sure if "identify" is the right word. debugLabels don't need to be unique unlike keys.